presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/CommandExecutor.java

```diff
@@ -13,23 +13,42 @@
  */
 package com.teradata.prestomanager.agent;
 
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
 import java.util.concurrent.TimeUnit;
+import java.util.stream.Collectors;
 
 import static java.lang.Runtime.getRuntime;
 import static java.lang.String.format;
 
-// TODO: Add logger
 public final class CommandExecutor
 {
+    private static final Logger LOGGER = LogManager.getLogger(CommandExecutor.class);
+
     private CommandExecutor() {}
 
     public static int executeCommand(String command)
             throws PrestoManagerException
     {
+        LOGGER.debug("Command to be executed: {}", command);
+
         try {
             Process process = getRuntime().exec(command);
+            String output = new BufferedReader(new InputStreamReader(process.getInputStream()))
```

Comment:
1. You don't close these
2. These will hang around until the process exits, which isn't good (And also makes the waitFor kind of useless since it's already waited).  Also, it will first do the stdout and then the stderr, which causes its own problems.  It's probably worth redirecting stderr to stdout so you don't need to worry about reading from both of them (They'll both get logged anyway, keeping them separate isn't so important).

You should use processBuilder to help you out here.